diff --git a/src/main/java/org/acme/AiHelper.java b/src/main/java/org/acme/AiHelper.java
index bb1d3fa..db9e4f3 100644
--- a/src/main/java/org/acme/AiHelper.java
+++ b/src/main/java/org/acme/AiHelper.java
@@ -4,22 +4,16 @@ import dev.langchain4j.service.SystemMessage;
 import dev.langchain4j.service.UserMessage;
 import io.quarkiverse.langchain4j.RegisterAiService;
 
-@RegisterAiService //(tools = Tools.class)
+//@RegisterAiService //(tools = Tools.class)
 public interface AiHelper {
 
-    @UserMessage("Hi, my name is {name}")
-    String greet(String name);
-
-    @UserMessage("What time is it?")
-    String time();
-
-    @SystemMessage("You are an assistant in a pet adoption centre. Your job is to help people find the right pet for them. Include descriptions of the specific individual to be adopted.")
-    @UserMessage("I would like to find a pet. Please suggest something unusual.  Please only suggest one.")
+//    @SystemMessage("You are an assistant in a pet adoption centre. Your job is to help people find the right pet for them. Include descriptions of the specific individual to be adopted.")
+//    @UserMessage("I would like to find a pet. Please suggest something unusual.  Please only suggest one.")
 //    @OutputGuardrails(AnimalValidator.class)
-    Pet pet();
-
-    @SystemMessage("You are an assistant in a pet adoption centre. Your job is to help people find the right pet for them. Include descriptions of the specific individual to be adopted.")
-    @UserMessage("I would like to find a pet which is {preference}. Please only suggest one.")
+//    Pet pet();
+//
+//    @SystemMessage("You are an assistant in a pet adoption centre. Your job is to help people find the right pet for them. Include descriptions of the specific individual to be adopted.")
+//    @UserMessage("I would like to find a pet which is {preference}. Please only suggest one.")
 //    @OutputGuardrails(AnimalValidator.class)
-    Pet pet(String preference);
+//    Pet pet(String preference);
 }
diff --git a/src/main/java/org/acme/BackendResource.java b/src/main/java/org/acme/BackendResource.java
index 84ee275..aa56b24 100644
--- a/src/main/java/org/acme/BackendResource.java
+++ b/src/main/java/org/acme/BackendResource.java
@@ -10,29 +10,16 @@ import jakarta.ws.rs.core.MediaType;
 @Path("/api")
 public class BackendResource {
 
-    @Inject
-    AiHelper ai;
-
-    @GET
-    @Produces(MediaType.TEXT_PLAIN)
-    public String hello(@QueryParam("name") String name) {
-        return ai.greet(name);
-    }
-
-    @Path("time")
-    @GET
-    @Produces(MediaType.TEXT_PLAIN)
-    public String time() {
-        return ai.time();
-    }
+//    @Inject
+//    AiHelper ai;
 
     @Path("pet")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     public Pet pet(@QueryParam("preference") String preference) {
         if (preference != null && !preference.isEmpty()) {
-            return ai.pet(preference);
+            return new Pet("","","","",0,"","",Animal.UNKNOWN);
         }
-        return ai.pet();
+        return new Pet("","","","",0,"","",Animal.UNKNOWN);
     }
 }
